### PR TITLE
Gidk 1z9fp8 dashboard inventory v1

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-profit-stocklist/dashboard-profit-stocklist.component.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        }
+        this.modalRef = this.modalService.show(
+            this.datepickerTemplate,
+            Object.assign({}, { class: 'modal-lg giddh-datepicker-modal', backdrop: false, ignoreBackdropClick: false })
+        );
+    }
+    /**
+     * This will hide the datepicker
+     *
+     * @memberof DaybookComponent
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-profit-stocklist/dashboard-profit-stocklist.component.ts:57]: Update the `@memberof` with current class through out the file
```suggestion
     * @memberof DashboardProfitStockList
```

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-purchase-report/dashboard-purchase-report.component.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        }
+        this.modalRef = this.modalService.show(
+            this.datepickerTemplate,
+            Object.assign({}, { class: 'modal-lg giddh-datepicker-modal', backdrop: false, ignoreBackdropClick: false })
+        );
+    }
+    /**
+     * This will hide the datepicker
+     *
+     * @memberof DaybookComponent
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-purchase-report/dashboard-purchase-report.component.ts:57]: Update `@memberof`

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-sales-report/dashboard-sales-report.component.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        }
+        this.modalRef = this.modalService.show(
+            this.datepickerTemplate,
+            Object.assign({}, { class: 'modal-lg giddh-datepicker-modal', backdrop: false, ignoreBackdropClick: false })
+        );
+    }
+    /**
+     * This will hide the datepicker
+     *
+     * @memberof DaybookComponent
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-sales-report/dashboard-sales-report.component.ts:57]: Update `@memberof`

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-selling-product/dashboard-selling-product.component.scss [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                    border-radius:10px 10px;
+                    margin: 7px 0;
+                }
+            }
+        }
+    }
+}
+}
+
+@media(max-width:575px){
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-selling-product/dashboard-selling-product.component.scss:86]: Do remove empty selectors

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-selling-product/dashboard-selling-product.component.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { BsDropdownModule } from 'ngx-bootstrap';
+@Component({
+    selector: 'dashboard-selling-product',
+    templateUrl: './dashboard-selling-product.component.html',
+    styleUrls: ['./dashboard-selling-product.component.scss'],
+
+})
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-selling-product/dashboard-selling-product.component.ts:10]: Remove empty lines

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-selling-product/dashboard-selling-product.component.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { BsDropdownModule } from 'ngx-bootstrap';
+@Component({
+    selector: 'dashboard-selling-product',
+    templateUrl: './dashboard-selling-product.component.html',
+    styleUrls: ['./dashboard-selling-product.component.scss'],
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-selling-product/dashboard-selling-product.component.ts:8]: Remove empty lines

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-activity/inventory-activity.component.scss [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        }
+    }
+    .right-active-data{
+        position: relative;
+        &:after{
+            position: absolute;
+            content: "";
+            width: 1px;
+            height: 100%;
+            background-color: #d9d9d9;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-activity/inventory-activity.component.scss:89]: Do use color vars

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-activity/inventory-activity.component.ts [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+import * as moment from 'moment/moment';
+import { GIDDH_DATE_FORMAT, GIDDH_NEW_DATE_FORMAT_UI } from '../../../../shared/helpers/defaultDateFormat';
+import { BreakpointObserver } from '@angular/cdk/layout';
+@Component({
+    selector: 'inventory-activity',
+    templateUrl: './inventory-activity.component.html',
+    styleUrls: ['./inventory-activity.component.scss'],
+
+})
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-activity/inventory-activity.component.ts:14]: Remove empty lines

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-dashboard.component.ts [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+@Component({
+    selector: 'inventory-dashboard',
+    templateUrl: './inventory-dashboard.component.html',
+    styleUrls: ['./inventory-dashboard.component.scss'],
+
+})
+
+export class InventoryDashboardComponent implements OnInit {
+
+    constructor(private _router: Router) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-dashboard.component.ts:12]: Remove `private _router: Router` as it is unused
```suggestion
    constructor() {
```

File: apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-dashboard.component.ts [hunk 10]
```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-dashboard.component.ts:2]: Remove `import { Router } from '@angular/router';`

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**